src/components/autocomplete/autocomplete.spec.js

```diff
@@ -27,6 +27,14 @@ describe('<md-autocomplete>', function() {
     return scope;
   }
 
+  function waitForVirtualRepeat(element) {
+    inject(function ($$rAF) {
+      $$rAF.flush();
+      element.scope().$apply();
+      $$rAF.flush();
+    });
```

Comment:
So, I think I get it: You just have to `scope.$apply()`, then, `$timeout.flush()`, then `$$rAF.flush()`, then again `scope.$apply()`, then finally `$$rAF.flush()` again and...BAM...virtual-repeat ready !!!
Nice !

<sub>(Some comments would be also nice here, but I guess that's not very straightforward.)</sub>